packages/dom/src/stripHTML.ts

```diff
@@ -16,7 +16,7 @@
 
 const buffer = document.createElement('div');
 
-export default function stripHTML(string) {
+export default function stripHTML(string: string) {
   // @todo: implement a cheaper way to strip markup.
   buffer.innerHTML = string;
   return buffer.textContent;
```

Comment:
`textContent` can be `null`. Should we force-return a string?

That would be relevant for places like this:

https://github.com/GoogleForCreators/web-stories-wp/blob/4a71cf6ec75cb952bb3868233268ede7550432fb/packages/story-editor/src/components/checklist/utils/characterCountForPage.js#L23-L27

Reply:
Yeah, that makes sense. Fixed in 1256c0c882.